python/ray/experimental/workflow/api.py

```diff
@@ -30,9 +27,10 @@ def step(func) -> WorkflowStepFunction:
 # Maybe we can also add a run_or_resume() call.
 
 
-def run(entry_workflow: Workflow,
-        storage: Optional[Union[str, Storage]] = None,
-        workflow_id: Optional[str] = None) -> ray.ObjectRef:
+def run(entry_workflow: "Workflow",
+        storage: "Optional[Union[str, Storage]]" = None,
+        workflow_id: Optional[str] = None,
+        share_fate_with_driver: bool = False) -> ray.ObjectRef:
```

Comment:
Can we remove this flag and always set it to False?

Alternatively, we should call it "detached", since it might not be the driver calling run. Actors or other tasks can start workflows.